errors/definitions.go

```diff
@@ -0,0 +1,135 @@
+	ErrMissingFieldSignature = models.SPVError{Message: "missing required field: signature", StatusCode: 400, Code: "error-missing-field-signature"}
+
+	// ErrMissingFieldPubKey is when the pubkey field is required but missing
+	ErrMissingFieldPubKey = models.SPVError{Message: "missing required field: pubkey", StatusCode: 400, Code: "error-missing-field-pubkey"}
+
+	// ErrMissingFieldSatoshis is when the satoshis field is required but missing
+	ErrMissingFieldSatoshis = models.SPVError{Message: "missing required field: satoshis", StatusCode: 400, Code: "error-missing-field-satoshis"}
+)
+
+// EMPTY FIELDS ERRORS
+var (
+	// ErrSenderHandleEmpty is when the server handle is empty
+	ErrSenderHandleEmpty = models.SPVError{Message: "empty sender handle", StatusCode: 400, Code: "error-empty-sender-handle"}
+
+	// ErrDtEmpty is when the dt field is empty
+	ErrDtEmpty = models.SPVError{Message: "empty dt", StatusCode: 400, Code: "error-empty-dt"}
+)
+
+// SPV ERRORS
+var (
+	// ErrNoOutputs is when there are no outputs
+	ErrNoOutputs = models.SPVError{Message: "invalid output, no outputs", StatusCode: 417, Code: "error-spv-no-outputs"}
+
+	// ErrNoInputs is when there are no inputs
+	ErrNoInputs = models.SPVError{Message: "invalid input, no inputs", StatusCode: 417, Code: "error-spv-no-inputs"}
+
+	// ErrInvalidParentTransactions is when the parent transactions are invalid
+	ErrInvalidParentTransactions = models.SPVError{Message: "invalid parent transactions, no matching transactions for input", StatusCode: 417, Code: "error-spv-invalid-parent-tx"}
+
+	// ErrLockTimeAndSequence is when the locktime and sequence are invalid
+	ErrLockTimeAndSequence = models.SPVError{Message: "nLocktime is set and nSequence is not max, therefore this could be a non-final tx which is not currently supported", StatusCode: 417, Code: "error-spv-locktime-sequence"}
+
+	// ErrOutputValueTooHigh is when the satoshis output is too high on a transaction
+	ErrOutputValueTooHigh = models.SPVError{Message: "invalid input and output sum, outputs can not be larger than inputs", StatusCode: 417, Code: "error-spv-output-value-too-high"}
+
+	// ErrBUMPAncestorNotPresent is when the input mined ancestor is not present in BUMPs
+	ErrBUMPAncestorNotPresent = models.SPVError{Message: "invalid BUMP - input mined ancestor is not present in BUMPs", StatusCode: 417, Code: "error-spv-bump-ancestor-not-present"}
+
+	// ErrBUMPCouldNotFindMindParent is when the mined parent for input could not be found
+	ErrBUMPCouldNotFindMindParent = models.SPVError{Message: "invalid BUMP - cannot find mined parent for input", StatusCode: 417, Code: "error-spv-bump-ancestor-not-present"}
+
+	// ErrNoMatchingTransactionsForInput is when no matching transaction for input can be found
+	ErrNoMatchingTransactionsForInput = models.SPVError{Message: "invalid parent transactions, no matching transactions for input", StatusCode: 417, Code: "error-spv-bump-ancestor-not-present"}
```

Comment:
not sure but i would use different code not same as 2 above

Reply:
Fixed